### PR TITLE
Proofread `changelogs/unreleased`

### PR DESCRIPTION
**Pull request by arctic-dreamer**
All changes are made according to the rules described in "[Contributing > How to write release notes](https://www.tarantool.io/en/doc/latest/contributing/release_notes/)".

This PR won't be merged till the end of March. We'll update it every week to add new changelog files.
When we proofread all files from `changelogs/unreleased`, the PR will be ready for review.

### CODE REVIEW

File: changelogs/unreleased/luarocks-disable-force-config.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 ## feature/luarocks
 
- * set FORCE_CONFIG=false for luarocks config to allow loading project-side .rocks/config-5.1.lua
+ * Set `FORCE_CONFIG=false` for luarocks config to allow loading project-side .rocks/config-5.1.lua.
```
Comment (Totktonada) [inline changelogs/unreleased/luarocks-disable-force-config.md:3]: While we're at it, it worth to drop the leading whitespace before an asterisk (at least for unification purposes).

Comment (arctic-dreamer) [reply changelogs/unreleased/luarocks-disable-force-config.md:3]: Sure, thanks for noticing!

File: changelogs/unreleased/add-check-for-user-input-of-the-number-of-iproto-threads.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
 ## bugfix/core
 
- * Added check for user input of the number of iproto threads - value
-   must be > 0 and less then or equal to 1000 (gh-6005).
+ * Added check for user input of the number of iproto threads — value
+   must be > 0 and less than or equal to 1000 (gh-6005).
```
Comment (Totktonada) [inline changelogs/unreleased/add-check-for-user-input-of-the-number-of-iproto-threads.md:4]: There is a thing around hyphen/dash, which doubts me.

There are technical texts, which aim direct reading (without any pre-rendering) as a primary or a second goal (say, markdown supports rendering, but easy to read directly). The most obvious example is a comment in code. All or most of text that are tightly connected to code are also of this kind: they are usually written as plain text, asciidoc, markdown. Here non-ASCII symbols are generally discouraged. Here we expect a hyphen (or two hyphens), where in book we would see a dash.

There are texts, which primarily aim some rendering tool, which will prettify it, inline hyperlinks and convert to a format of a reader tools (generate HTML, PDF and so on). Those are texts for modern websites for example. Here we use Unicode en/em dash symbols or a special syntax to represent them using ASCII.

I feel release notes as a text of the first kind. Is the neural network in my head wrong here? What do you think?

Comment (xuniq) [reply changelogs/unreleased/add-check-for-user-input-of-the-number-of-iproto-threads.md]: I agree. So I am going to apply the rule from the doc [contribution guide](https://www.tarantool.io/ru/doc/latest/contributing/docs/style/#dashes) and use "---" as a dash. The documentation builder renders this character sequence in the source into em dash.

File: changelogs/unreleased/drop-net-box-console.md [hunk 3]
```diff
@@ -1,8 +1,4 @@
 ## feature/core
 
-* **[Breaking change]** net.box console support, which was marked deprecated
+* **[Breaking change]** `net.box` console support, which was marked deprecated
   in 1.10, was dropped. Use `require('console').connect()` instead.
-
-----
-
-Breaking change: net.box console support was dropped.
```
Comment (Totktonada) [inline changelogs/unreleased/drop-net-box-console.md:8]: It is the note for a release manager. Please, keep it.

**Comment by Totktonada** [comment]
The patchset should meet several requirements to land into the `master` branch:

* Follow [our commit message guidelines](https://www.tarantool.io/en/doc/latest/dev_guide/developer_guidelines/#how-to-write-a-commit-message). Take care to 50/72 rules (lines length) and commits atomicity[^1]. Don't forget to add an appropriate prefix for commit titles: `doc:` is okay here.
* Fix all errors reported by the `lint / checkpatch` CI job. Take care to `NO_DOC`, `NO_TEST`, `NO_CHANGELOG` directives (look at examples in git log).
* If you aim to fix changelogs, which lands into `master` before a particular commit, please, point this commit. So I'll know what exactly I should review.

[^1]: I would like to see formatting/wording/meaning changes as separate commits, but I don't insist if it requires a lot of effort.

File: changelogs/unreleased/drop-net-box-console.md (hunk 3 above)
Comment (Totktonada) [reply changelogs/unreleased/drop-net-box-console.md:8]: I suggest to run `./tools/gen-release-notes` to look at the resulting release notes. In particular, it'll show such release manager notes separately to highlight them somewhere at top.

File: changelogs/unreleased/gh-5385-tiny-tuples-v3.md [hunk 4]
```diff
@@ -1,3 +1,4 @@
 ## feature/core
 
-* Introduce compact tuples that allow to save 4 bytes per tuple in case of small userdata  (gh-5385)
+* Introduce compact tuples that allow saving 4 bytes per tuple
+  in case of small userdata (gh-5385).
```
Comment (Totktonada) [inline changelogs/unreleased/gh-5385-tiny-tuples-v3.md:4]: Nit: It is better to don't make text much shorter than 80 symbols to make the entire changelog look visually same.

File: changelogs/unreleased/gh-5799-add-info-to-fiber-object.md [hunk 5]
```diff
@@ -1,4 +1,6 @@
 ## feature/lua/fiber
 
-* Introduce `fiber_object:info()` to get `info` from fiber. Works as `require(fiber).info()` but only for one fiber.
+* Introduce `fiber_object:info()` to get `info` from fiber.
+  Works as `require(fiber).info()` but only for one fiber.
```
Comment (Totktonada) [inline changelogs/unreleased/gh-5799-add-info-to-fiber-object.md:4]: require(fiber) -> require('fiber')

File: changelogs/unreleased/gh-5941-datetime-type-support.md [hunk 6]
```diff
@@ -1,8 +1,10 @@
...
-   or rfc3339 formats (gh-6731);
- * The range of supported years has been extended in all parsers to cover
+ * Added a new builtin module `datetime.lua` that allows to operate
+   timestamps and intervals values (gh-5941).
+
+ * Parsed method to allow converting string literals in extended iso-8601
+   or rfc3339 formats (gh-6731).
+
+ * Extended the range of supported years in all parsers to cover
    fully -5879610-06-22..5879611-07-11 (gh-6731).
```
Comment (Totktonada) [inline changelogs/unreleased/gh-5941-datetime-type-support.md:10]: Those bullets are still indented with a whitespace.

File: changelogs/unreleased/gh-6043-fiber-wakeup-self.md [hunk 7]
```diff
@@ -1,12 +1,14 @@
...
 * **[Breaking change]** `fiber.wakeup()` in Lua and `fiber_wakeup()` in C became
-  NOP on the currently running fiber. Previously they allowed to "ignore" the
-  next yield or sleep leading to unexpected spurious wakeups. Could lead to a
-  crash (in debug build) or undefined behaviour (in release build) if called
+  NOP on the currently running fiber.
+
+  Previously, they allowed to "ignore" the
+  next yield or sleep, leading to unexpected spurious wake-ups. Could lead to a
+  crash (in debug build) or undefined behavior (in release build) if called
   right before `fiber.create()` in Lua or `fiber_start()` in C (gh-6043).
```
Comment (Totktonada) [inline changelogs/unreleased/gh-6043-fiber-wakeup-self.md:9]: Lines breaks are a bit strange here.

File: changelogs/unreleased/fix-undeleted-unix-socket-path-and-multiple-socket-closing.md [hunk 8]
```diff
@@ -1,6 +1,5 @@
 ## bugfix/core
 
- * Fixed error, related to the fact, that if user changed listen address,
-   all iproto threads closed same socket multiple times.
-   Fixed error, related to the fact, that tarantool not deleting the unix
-   socket path, when it's finishing work.
+* Fixed error related to the fact that if a user changed the listen address,
+  all iproto threads closed the same socket multiple times.
+  Fixed error related to tarantool not deleting the unix socket path when finishing work.
```
[the remark this answers is not in the document]
Comment (xuniq) [reply changelogs/unreleased/fix-undeleted-unix-socket-path-and-multiple-socket-closing.md:5]: I have split them into separate bullets

File: changelogs/unreleased/gh-5819-election-triggers.md [hunk 9]
```diff
@@ -1,5 +1,5 @@
 ## feature/replication
 
 * Introduce on_election triggers. The triggers may be registered via
```
Comment (Totktonada) [inline changelogs/unreleased/gh-5819-election-triggers.md:3]: (Same here.)

File: changelogs/unreleased/gh-5882-log-levels.md [hunk 10]
```diff
@@ -1,6 +1,6 @@
 ## feature/lua/log
 
-* Implemented support of symbolic log levels representation
+* Implement support of symbolic log levels representation
```
Comment (Totktonada) [inline changelogs/unreleased/gh-5882-log-levels.md]: (Same here.)

File: changelogs/unreleased/gh-5941-datetime-type-support.md [hunk 11]
```diff
@@ -1,10 +1,10 @@
 ## feature/lua/datetime
 
- * Added a new builtin module `datetime.lua` that allows to operate
+ * Add a new builtin module `datetime.lua` that allows to operate
    timestamps and intervals values (gh-5941).
```
Comment (Totktonada) [inline changelogs/unreleased/gh-5941-datetime-type-support.md]: (Same here, AFAIU we should use past simple.)

File: changelogs/unreleased/gh-5941-datetime-type-support.md [hunk 12]
```diff
@@ -1,10 +1,10 @@
...
 
- * Parsed method to allow converting string literals in extended iso-8601
+ * Parse method to allow converting string literals in extended iso-8601
    or rfc3339 formats (gh-6731).
```
Comment (Totktonada) [inline changelogs/unreleased/gh-5941-datetime-type-support.md]: **Added** the methods?

File: changelogs/unreleased/gh-5941-datetime-type-support.md [hunk 13]
```diff
@@ -1,10 +1,10 @@
...
 
- * Extended the range of supported years in all parsers to cover
+ * Extend the range of supported years in all parsers to cover
    fully -5879610-06-22..5879611-07-11 (gh-6731).
```
Comment (Totktonada) [inline changelogs/unreleased/gh-5941-datetime-type-support.md]: (Same, should be past simple in my understanding.)

File: changelogs/unreleased/gh-6131-fix-crash-update-conflict.md [hunk 14]
```diff
@@ -1,3 +1,3 @@
 ## bugfix/core
 
-* Fixed a crash in MVCC during simultaneous update of a key in different transactions (gh-6131)
+* Fixed a crash in MVCC during simultaneous update of a key in different transactions (gh-6131).
```
Comment (Totktonada) [inline changelogs/unreleased/gh-6131-fix-crash-update-conflict.md]: Nit: long line.

File: changelogs/unreleased/gh-6266-fix-crash-in-lua-triggers.md [hunk 15]
```diff
@@ -1,5 +1,5 @@
 ## bugfix/lua/triggers
 
-* Fixed possibility crash in case when trigger removes itself.
-  Fixed possibility crash in case when someone destroy trigger,
-  when it's yield (gh-6266).
\ No newline at end of file
+* Fixed the possibility of a crash in case when trigger removes itself.
+  Fixed the possibility of a crash in case someone destroys trigger
+  when it's yielding (gh-6266).
```
Comment (Totktonada) [inline changelogs/unreleased/gh-6266-fix-crash-in-lua-triggers.md:5]: No empty line between paragraphs.

File: changelogs/unreleased/gh-6396-transaction-public-id.md [hunk 16]
```diff
@@ -1,4 +1,4 @@
 ## feature/core
 
-* Added new `box.txn_id()` function that returns the id of the current transaction
+* Add the new `box.txn_id()` function that returns the id of the current transaction
   if called within a transaction, nil otherwise.
```
Comment (Totktonada) [inline changelogs/unreleased/gh-6396-transaction-public-id.md]: (In my understanding it should be in past simple.)

File: changelogs/unreleased/add-libunwind-submodule.md [hunk 17]
```diff
@@ -1,9 +1,9 @@
...
 
-* Added bundling of _GNU libunwind_ to support backtrace feature on
+* Add bundling of _GNU libunwind_ to support backtrace feature on
   _AARCH64_ architecture and distributives that don't provide _libunwind_
   package.
 
-* Re-enabled backtrace feature for all _RHEL_ distributions by default, except
+* Re-enable backtrace feature for all _RHEL_ distributions by default, except
   for _AARCH64_ architecture and ancient _GCC_ versions, which lack compiler
   features required for backtrace (gh-4611).
```
Comment (Totktonada) [inline changelogs/unreleased/add-libunwind-submodule.md:8]: (Here too, should be past simple IMHO.)

File: changelogs/unreleased/gh-6789-fix-memleak-in-vdbe.md [hunk 18]
```diff
@@ -1,3 +1,3 @@
 ## bugfix/sql
 
-* Fixed memleak in SQL during calling of user-defined function (gh-6789).
+* Fixed memory leak in SQL during calling of user-defined function (gh-6789).
```
Comment (Totktonada) [inline changelogs/unreleased/gh-6789-fix-memleak-in-vdbe.md:3]: *a* memory leak?

File: changelogs/unreleased/box-runtime-info-tuple.md [hunk 19]
```diff
@@ -1,7 +1,7 @@
 ## feature/lua
 
-* Add `box.runtime.info().tuple` metric to track amount of memory occupied by
+* Add `box.runtime.info().tuple` metric to track the amount of memory occupied by
   tuples allocated on runtime arena (gh-5872).
```
Comment (Totktonada) [inline changelogs/unreleased/box-runtime-info-tuple.md]: (*Added*)

File: changelogs/unreleased/fiber-parent-backtrace.md [hunk 20]
```diff
@@ -1,5 +1,5 @@
 ## feature/core
 
-* Implemented collection of parent backtrace for newly created fibers.
-  To enable the feature, call `fiber.parent_backtrace_enable` — to disable it
+* Implement collection of parent backtrace for the newly created fibers.
+  To enable the feature, call `fiber.parent_backtrace_enable`. To disable it,
   call `fiber.parent_backtrace_disable`: disabled by default (gh-4302).
```
Comment (Totktonada) [inline changelogs/unreleased/fiber-parent-backtrace.md:5]: (The same comment regarding past simple is applicable here too.)

File: changelogs/unreleased/gh-5815-memprof-enrich-symtab.md [hunk 21]
```diff
@@ -1,7 +1,8 @@
...
-  is created. As a result memory profiler parser can enrich its symbol table
-  with the new functions and traces (gh-5815). Furthermore, there are symbol
-  generations introduced within internal parser structure to handle possible
-  collisions of function addresses and trace numbers.
+* Now the memory profiler emits events of the new type when a function or a trace
+  is created. As a result the memory profiler parser can enrich its symbol table
+  with the new functions and traces (gh-5815).
+
+  Furthermore, there are symbol generations introduced within the internal parser structure
+  to handle possible collisions of function addresses and trace numbers.
```
Comment (Totktonada) [inline changelogs/unreleased/gh-5815-memprof-enrich-symtab.md]: Too long line.

File: changelogs/unreleased/gh-6043-fiber-wakeup-self.md [hunk 22]
```diff
@@ -4,11 +4,11 @@
   NOP on the currently running fiber.
 
   Previously, they allowed to "ignore" the
-  next yield or sleep, leading to unexpected spurious wake-ups. Could lead to a
+  next yield or sleep, leading to unexpected spurious wake-ups. This could lead to a
   crash (in debug build) or undefined behavior (in release build) if called
   right before `fiber.create()` in Lua or `fiber_start()` in C (gh-6043).
```
Comment (Totktonada) [inline changelogs/unreleased/gh-6043-fiber-wakeup-self.md]: Unusual line breaks. I don't know what a text editor you're using, but I use `gqap` to format a paragraph in Vim.

File: changelogs/unreleased/gh-6060-enable-backtrace-feature-on-aarch64.md [hunk 23]
```diff
@@ -1,3 +1,3 @@
 ## feature/core
 
-* Added support for backtrace feature on AARCH64 architecture (gh-6060).
+* Add support for backtrace feature on AARCH64 architecture (gh-6060).
```
Comment (Totktonada) [inline changelogs/unreleased/gh-6060-enable-backtrace-feature-on-aarch64.md:3]: (I vote for past simple.)

File: changelogs/unreleased/gh-6150-memtx-mvcc-memory-monitoring.md [hunk 24]
```diff
@@ -1,3 +1,3 @@
 ## feature/memtx
 
-* Introduced memtx mvcc memory monitoring (gh-6150).
+* Introduce memtx MVCC memory monitoring (gh-6150).
```
Comment (Totktonada) [inline changelogs/unreleased/gh-6150-memtx-mvcc-memory-monitoring.md:3]: (I vote for past simple.)

File: changelogs/unreleased/gh-6754-term-bump-before-promote.md [hunk 25]
```diff
@@ -1,6 +1,6 @@
 ## feature/raft
 
-* Added `term` field to `box.info.synchro.queue`. It contains term of the
-  last PROMOTE. It is usually equal to `box.info.election.term` but may be
-  less than election term when new round of elections started, but no one
+* Add `term` field to `box.info.synchro.queue`. It contains term of the
```
Comment (Totktonada) [inline changelogs/unreleased/gh-6754-term-bump-before-promote.md:3]: (I vote for past simple.)

File: changelogs/unreleased/gh-6930-isolation-level.md [hunk 26]
```diff
@@ -1,3 +1,3 @@
 ## feature/core
 
-* Introduced transaction isolation levels in Lua and IPROTO (gh-6930).
+* Introduce transaction isolation levels in Lua and IPROTO (gh-6930).
```
Comment (Totktonada) [inline changelogs/unreleased/gh-6930-isolation-level.md:3]: (I vote for past simple.)

File: changelogs/unreleased/introduce-arm64-preliminary-support.md [hunk 27]
```diff
@@ -1,8 +1,8 @@
 ## feature/luajit
 
-* Introduced preliminary support of GNU/Linux ARM64 and MacOS M1. In the scope of
+* Introduce preliminary support of GNU/Linux ARM64 and MacOS M1. In the scope of
   this activity, the following issues have been resolved:
-  - Introduced support for a full 64-bit range of lightuserdata values (gh-2712).
+  - Introduce support for a full 64-bit range of lightuserdata values (gh-2712).
```
Comment (Totktonada) [inline changelogs/unreleased/introduce-arm64-preliminary-support.md:5]: (I vote for past simple.)

File: changelogs/unreleased/update-libicu-static-build.md [hunk 28]
```diff
@@ -1,3 +1,3 @@
 ## feature/build
 
-* Updated libicu version to 71.1 for static build.
+* Update `libicu` version to 71.1 for static build.
```
Comment (Totktonada) [inline changelogs/unreleased/update-libicu-static-build.md:3]: (I vote for past simple.)

File: changelogs/unreleased/gh-6773-datetime-sql-builtin-functions.md [hunk 29]
```diff
@@ -1,3 +1,3 @@
 ## feature/sql
 
-* SQL built-in functions NOW() and DATE_PART() were introduced (gh-6773).
+* Introduce SQL built-in functions `NOW()` and `DATE_PART()` (gh-6773).
```
Comment (Totktonada) [inline changelogs/unreleased/gh-6773-datetime-sql-builtin-functions.md:3]: (I vote for past simple.)

File: changelogs/unreleased/elections-pre-vote.md [hunk 30]
```diff
@@ -1,15 +1,15 @@
...
+* Servers with elections enabled won't start new elections as long as at least
+  one of their peers sees the current leader. They also won't start the elections
+  when they don't have a quorum of connected peers. This should reduce cases
+  when a server which has lost connectivity to the leader disrupts the whole
+  cluster by starting new elections (gh-6654).
 
- * Added `leader_idle` field to `box.info.election` table. This value shows time
-   in seconds since the last communication with a known leader (gh-6654).
+* Add the `leader_idle` field to `box.info.election` table. The value shows time
+  in seconds since the last communication with a known leader (gh-6654).
```
Comment (Totktonada) [inline changelogs/unreleased/elections-pre-vote.md]: (I vote for past simple.)

**Review by Totktonada — COMMENTED** [review]
The changes are generally good for me. I'm waiting for fixes and squashing changes.

I'll no re-verify it, so please resolve my comments once they're fixed.

**Review by Totktonada — APPROVED** [review]

**Pull request merged**